frontend/src/components/SelectDocuments.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react'
+import { Selector } from './Selector'
+import { getAllDocumentClasses } from '../utils/ApiWrapper'
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+import DatePicker from 'react-datepicker'
+import 'react-datepicker/dist/react-datepicker.css'
+import 'react-datepicker/dist/react-datepicker-cssmodules.css'
+
+const mapStateToProps = state => ({
+  isPM: state.user.isPM
+})
+
+const mapDispatchToProps = dispatch => {
+  return bindActionCreators(
+    {
+      //put actions here
+    },
+    dispatch
+  )
+}
+class SelectDocumentsPage extends React.Component {
+  constructor(props) {
+    super(props)
+    var today = new Date()
+    this.state = {
+      // all docClasses
+      docClass: {},
+      // docClasses filtered from docClasses using query
+      filtered: {},
+      // due date to be set by user so that it can be passed on
+      DueDate: today,
```

Comment:
Is the today variable coming from the date-picker react library? If so it might be beneficial to mention that in a comment. Maybe something like

// due date to be set by user so that it can be passed on, set to today (from date-picker)

Reply:
Changes Made.